.github/workflows/testing-and-deployment.yml

```diff
@@ -25,7 +25,7 @@ jobs:
     name: Mac OS Unit Testing
     strategy:
       matrix:
-        python-version: ['3.7', '3.8']
+        python-version: ['3.7', '3.10']
```

Comment:
I'd prefer the structure in #2837 or similar, with the release candidate specifically defined outside the matrix.  Otherwise, it isn't going to be obvious to contributors what the subtleties are in the tests.  But it could happen here rather than #2837.